types/fbinstant/index.d.ts

```diff
@@ -0,0 +1,223 @@
+// Type definitions for FBInstant 6.1
```

Comment:
`fbinstant` appears to be an unrelated npm package name. To avoid name collisions and to follow conventions set by other typings, could you rename to `facebook-instant-games`?

Reply:
Just to be sure, i should rename the folder in the types directory as well right?

Reply:
Yep that's what I'm suggesting. Thanks!